src/material/autocomplete/_autocomplete-theme.scss

```diff
@@ -10,7 +10,12 @@
   @if inspection.get-theme-version($theme) == 1 {
     @include _theme-from-tokens(inspection.get-theme-tokens($theme, base));
   }
-  @else {}
+  @else {
+    @include sass-utils.current-selector-or-root() {
+      @include token-utils.create-token-values(tokens-mat-autocomplete.$prefix,
```

Comment:
I think Miles wants us to also do this for the other theming mixins even if they're not currently used. I see we have it for `base` and `color`, but it's missing for `typography` and `density`.